community/server/src/docs/ops/server-configuration.asciidoc

```diff
@@ -3,15 +3,14 @@
 
 .Quick info
 ***********
-* The server's primary configuration file is found under _conf/neo4j-server.properties_.
-* Low-level performance tuning parameters and configuration of legacy indexes and the remote shell are found in _conf/neo4j.properties_.
+* Neo4j's configuration is found in _conf/neo4j.properties_.
```

Comment:
That's a good point. There's a far more general point there. I'll treat it as orthogonal to this change and bring it up with the Docs team.